### DISCUSSION TITLE
External SCSI drives not recognised

### DISCUSSION THREAD

**Discussion by mg-man** [erichelgeson/BlueSCSI-v1#43] [Q&A] [answered]
I have just replaced the internal drive in my SE/30 with a BlueSCSI (1.0c), but now can't access any of my external HDs.  I have tried various drives and termination configs, but the drives never show up in SCSIprobe, etc.  If I disconnect the BlueSCSI, all is well.  I have also tried leaving the BlueSCSI connected but with the termination jumpers removed - depending on the termination config if the external drive, the Mac either hangs during boot or throws up the flashing "floppy" icon - it never fully boots.  Has anyone used external SCSI devices with a BlueSCSI?  Suggestions?

**Comment by mg-man** [comment]
OK... so after spending hours in SCSI h*ll... I now have things working.  I'm not able to put my finger on exactly what was causing the issue... need to investigate some more.  What initially got things going was 'moving' the BlueSCSI from Device ID 0 to ID 6.  When I did this, I was able to boot off my external HD... *and* the BlueSCSI mounted as well.  I *thought* there might be some strange behavior with the BlueSCSI set to ID 0, but when I set the BlueSCSI to be the startup drive (still on ID 6) and restarted, that worked fine, ***BUT*** my external drive was again MIA (SCSIprobe could not see it).  After spending **_way_** too many hours trying different termination settings, IDs, and re-initializing the BlueSCSI partition with Apple HD Setup... I eventually got back to a configuration that is working - with BlueSCSI on ID 0 and my external HD on ID 1.  I did end up setting my external HD to supply Term PWR... whether that was the 'trick' I'm not sure... like I said, I need to do some more testing...

**Comment by mg-man** [comment, marked as answer]
More testing has revealed the issue... the key to the puzzle comes from my discovery above that _'moving' the BlueSCSI from Device ID 0 to ID 6_ got things going. Further trial and error led me to discover that the problem was nothing to do with the BlueSCSI (whew, right!?)... but related to the fact that the drive I was using externally was _ignoring_ the SCSI ID jumpers and **always coming up as ID 0!!**

The drive is a modern(ish) Quantum Atlas V 18.3Gb, so I have to use an SCA80 <> 50pin adapter. I have three different types of the latter, two versions (with termination) made by **max1zzz** from 68kMLA and another no-name (no termination) Chinese one. The drive works fine with all of them... but in ALL cases, it completely ignores the SCSI ID jumpers on the adapter!! [FWIW, I have an older 4.3Gb SCA80 Quantum that picks up the ID setting just fine!]

So... I'll mark this as answered... and will do some more investigation to see if there is some kind of firmware setting (there are NO jumpers on this drive) that forces the _ID 0 only_ behaviour. :-|

**Reply by erichelgeson** [reply]
Glad you got it working!
